### DISCUSSION TITLE
How to add shortcut in Mac Ventura

### DISCUSSION THREAD

**Discussion by laser13** [rcloran/lr-inaturalist-publish#10] [General]
https://community.adobe.com/t5/lightroom-classic-discussions/custom-menu-shortcuts-have-dropped-off/td-p/13436136
<img width="450" alt="image" src="https://github.com/rcloran/lr-inaturalist-publish/assets/1689628/bb1e4e03-4f69-4a52-be49-2895a0a56e11">
Need to add 3 spaces in start
```
   Group photos into observation
```

**Comment by rcloran** [comment]
Correct. I have noted this is a footnote of the README; if you have any suggestions on how to make this clearer I'd be happy to review a pull request or implement suggestions you describe.
